tests/test_ingest.py

```diff
@@ -3,40 +3,43 @@
 These tests verify that various matrix file types can be extracted and
 transformed, as expected by code that loads transformed data into Firestore.
 
-Test doubles are used for test speed and isolation.
+Mocks are used for test speed and isolation.
```

Comment:
It looks like the fresh content in `master` was overwritten by outdated content in your branch.  

In other words, the upstream version of `test_ingest.py` (left, red) is what was updated last week to account for removing the Firestore emulator (#44), plus Jean's updates testing gzipped files (#45).  The version you've committed here reverts all that and is outdated.

Quick fix: replace all the content in this file with that from `master`, i.e. copy https://raw.githubusercontent.com/broadinstitute/scp-ingest-pipeline/0b6cb898ab9f473e1b654876cdc0f552518e92b1/tests/test_ingest.py and replace this file with that.